### ISSUE TITLE
Sweep across learning rates

### ISSUE THREAD

**Issue by mosscoder** [mosscoder/mpg-horses#47]
We'll first ballpark a learning rate, returning to it later during fine tuning:

Ballpark learning rates to test:

```[0.01, 0.001, 0.0001, 0.00001]```

**Comment by samsoe** [comment]
Best learning rate: 0.0001
* [learning_rate_histogram](https://github.com/mosscoder/mpg-horses/blob/main/results/figures/learning_rate_histogram.png)
* ![Image](https://github.com/user-attachments/assets/3972bbb1-d920-4d61-a65d-7d54134f5d2b)

**Comment by samsoe** [comment]
Commit hash: 7a825e2063dca4ebd9999dae00651a7758be7839
